src/main/java/com/aws/greengrass/clientdevices/auth/api/GetClientDeviceAuthTokenEvent.java

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
This should probably belong in the session domain. Follow the directory structure we're using for the other domain events (take the term quite literally -- a domain event is an event that a software domain emits)